mantidproject/mantid
# Vesuvio - LoadVesuvio load monitor property

LoadVesuvio algorithm now has the option to load the monitor data in addition to it's normal operation.

**To test:**
- Ensure all the new test cases in LoadVesuvioTest (system test) run and pass.
- Running the following script should produce the expected data from `LoadVesuvio` as well as an additional workspace that contains the monitor data:

```
EVS21000_21005 = LoadVesuvio('21000-21005', '3-134', SumSpectra=True, LoadMonitors=True)
```
- Ensure that there are 2 workspaces produced in the ADS
- They should be named `EVS21000_21005` and `EVS21000_21005_monitors`
- Ensure the `_monitors` workspace contains 2 spectra 
  - When these are plotted they should look like: 
    ![image](https://cloud.githubusercontent.com/assets/13132128/14245236/c44f797a-fa57-11e5-9d73-01716665478d.png)

Fixes #14992

[Release Notes](https://github.com/mantidproject/mantid/blob/14992_LoadVesuvio_LoadMonitor_Property/docs/source/release/v3.7.0/indirect_inelastic.rst#id28)

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
